.autorc

```diff
@@ -0,0 +1,68 @@
+{
+  "plugins": [
+    "git-tag",
+    "all-contributors",
+    "conventional-commits",
+    "first-time-contributor",
+    "released"
+  ],
+  "release": {
+    "prerelease": true
+  },
+  "prereleaseBranches": [
+    "master"
```

Comment:
you might be interested in [this](https://intuit.github.io/auto/pages/generated/shipit.html#without-%22next%22-branch-(--only-graduate-with-release-label)

Reply:
Thank you!
Wouldn't that unable us to automatically publish a release after every merged PR though?
From what I understand, that would trigger a new release only when the label `release` is added to the PR.
Our intentions are a bit different: we are an open development project, that wants to release as soon as there is something new, even if we're still in beta production stage.

The "prereleaseBranches" set on master is temporary, until we get out of beta stage.
Then we will remove it.